src/Content.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const Content = styled.main`
+  width: 360px;
+  padding: 40px 0 20px;
+  margin: 0 auto auto;
+  overflow-y: auto;
+`;
+
+export default Content;
```

Comment:
Well that must be right cause we kinda did this together 😂

Reply:
Haha, yes, thank you so much! Justus, the master of grid ;)